liblwgeom/liblwgeom_internal.h

```diff
@@ -122,6 +122,14 @@
 #define SIZE_SET(varsize, size) (((varsize) & 0x00000003)|(((size) & 0x3FFFFFFF) << 2 ))
 
 /**
+* Macro that returns:
+* -1 if n < 0,
+*  1 if n > 0,
+*  0 if n == 0
+*/
+#define SIGNUM(n) (n > 0) - (n < 0)
```

Comment:
`SIGNUM(10) * 5` will become `(10 > 0) - (10 < 0) * 5`. Maybe another parentheses?

Reply:
You are totally right, I'll update it.

Reply:
it bothers me that no test failed, although there are some maths in this PR affected by this.

Reply:
It was never used as part of an math operation. I've added a couple of extra tests.